src/cmd/api/dto/json/jsonDTO.go

```diff
@@ -41,3 +41,20 @@ func VideoToStatusJson(video *models.Video) VideoStatus {
 
 	return videoStatus
 }
+
+// LinkJson DTO
+type LinkJson struct {
+	Rel    string `json:"rel" exemple:"status"`
+	Href   string `json:"href" exemple:"/api/v1/videos/{id}/status"`
+	Method string `json:"method" exemple:"get"`
```

Comment:
Should be "GET" instead of "get" since you write 
 			Method: "GET",
in src/cmd/api/controllers/video_upload_handler.go l. 267